collector/rpc_metrics.go

```diff
@@ -39,11 +50,15 @@ func NewNodeRpcMetrics(
 	accountId string) *NodeRpcMetrics {
 
 	return &NodeRpcMetrics{
-		accountId:                accountId,
-		internalClient:           internalClient,
-		externalClient:           externalClient,
-		validatorRPCFailCount:    0,
-		validatorRPCSuccessCount: 0,
+		accountId:                     accountId,
+		internalClient:                internalClient,
+		externalClient:                externalClient,
+		validatorRPCFailCount:         0,
+		validatorRPCSuccessCount:      0,
+		internalStatusRPCFailCount:    0,
+		internalStatusRPCSuccessCount: 0,
+		externalStatusRPCFailCount:    0,
+		externalStatusRPCSuccessCount: 0,
```

Comment:
Pretty sure these should be defaulting to `0` anyhow, I don't think you need to set them in your struct declaration

Reply:
they need to be instantiated on the object or its a nil ref